index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aghanistan Developer Confrenece</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/vsn5col.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-menu">
+            <a  class="icon" href="javascript:void(0);"><img class="nav-menu-img" src="./img/burgermenu.png"></a>
+            <div class="top-nav">
+                <ul id="myLinks-up" class="top-nav-list">
+                    <li><a href="#">Logout</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#"><img src="./img/twitter.png" alt=""></a></li>
+                    <li><a href="#"><img src="./img/facebook.png" alt=""></a></li>
+                </ul>
+            </div>
+            <div class="bottom-nav">
+                <ul id="myLinks" class="top-menu-list">
+                    <li><a href="#works">Afghan Dev</a></li>
+                    <li><a href="#aboutme">News</a></li>
+                    <li><a href="#sponser">Sponser</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="./about.html">About</a></li>
+                </ul>
+                <div class="nav-branding">
+                    <a href="./index.html"><img src="./img/afghan-logo.png" alt=""></a>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="headline-section">
+            <div class="headline-container">
+                <p class="welcome-text">"Hello! Afghanistan Developer"</p>
+                <h1 class="headline-title">Afghanistan Developers National Summit 2022</h1>
+                <p class="headline-content">It is a long established fact that a reader will be distracted 
+                    by the readable content of a page when looking at its layout.
+                    </p>
+                <h2 class="headline-date">2022.12.15(THU) ~ 16(FRI)</h2>
+                <p class="headline-last-p">@National Museum of Afghanistan, Art Center and More</p>
+            </div>
+        </section>
+        <section class="program-section">
+            <h3>Main Program</h3>
```

Comment:
- [ ] The middle section should include a section header, a divider, five individual articles, and a line of text at the bottom, kindly consider adding the divider below the section header

|Your page|Provided design|
|-|-|
|![Screenshot_20221203_093057](https://user-images.githubusercontent.com/25227022/205432035-61320a51-d147-4019-81ec-414b3e39ca09.png)|![Screenshot_20221203_093159](https://user-images.githubusercontent.com/25227022/205432041-b89eb2f3-e123-4c2e-ba30-2b807f3c0c3b.png)|